oar/core/config_store.py

```diff
@@ -0,0 +1,135 @@
+import requests
+import json
+import yaml
+import os
+import oar.core.util as util
+from oar.core.exceptions import ConfigStoreException
+from requests.exceptions import RequestException
+from yaml import YAMLError
+
+
+class ConfigStore:
+    """
+    Config Store object will be used in other modules to retrieve
+    release data and app level config settings. e.g. advisories, builds etc.
+
+    cs = ConfigStore("4.12.11")
+    ads = cs.get_advisories()
+
+    """
+
+    def __init__(self, release):
+        if release == "":
+            raise ConfigStoreException("argument release is required")
+
+        if not util.is_valid_z_release(release):
+            raise ConfigStoreException(f"invalid zstream release format {release}")
+
+        self.release = release
+
+        # load local config file
+        path = os.path.dirname(__file__) + "/config_store.json"
+        with open(path) as f:
+            self._local_conf = json.load(f)
+
+        # download ocp build data with release branch e.g. openshift-4.12
+        branch = "openshift-%s" % util.get_y_release(self.release)
+        url = self._local_conf["build_data_url"] % branch
```

Comment:
@rioliu-rh just warning: our solution has one high risk. once the url does not work (I mean there is no such data), our whole process are blocked.
so, we could consider if there is other way to get the data we want in the future.

Reply:
@TODO, if there is no such branch found on ocp-build-data, need to find a fallback solution get the same data